build/media_source/plg_system_shortcut/js/shortcut.es6.js

```diff
@@ -103,11 +103,11 @@
     let dl = '<dl>';
     dlItems.forEach((titles, shortcut) => {
       dl += '<dt><kbd>J</kbd>';
-      shortcut.split('+').forEach((key) => {
-        dl += ` <kbd>${key.trim()}</kbd>`;
+      shortcut.split('+').forEach(key => {
```

Comment:
```suggestion
      shortcut.split('+').forEach((key) => {
```
Fix JS code style error reported by drone.